test-data/configuration_tests/input/jobs/snake_oil_diff.py

```diff
@@ -3,22 +3,16 @@
 
 
 def writeDiff(filename, ecl_sum, key1, key2):
-    with open(filename, "w") as f:
+    with open(filename, "w", encoding="utf-8") as f:
         for v1, v2 in zip(ecl_sum.numpy_vector(key1), ecl_sum.numpy_vector(key2)):
             diff = v1 - v2
-            f.write("%f\n" % diff)
+            f.write(f"{diff}\n")
```

Comment:
I know that you like the f-formatting, but is it really producing the same result? As in the old one we set precision to float while with the new  we let the type to be decided by numpy.

Reply:
Will change to `f"{diff:f}"`